### DISCUSSION TITLE
Add aarch64 as a configuration

### DISCUSSION THREAD

**Discussion by henriksod** [mvukov/rules_ros2#150] [Ideas]
Once this issue https://github.com/mvukov/rules_ros2/issues/144 is resolved, I suggest we add aarch64 as a configurable target architecture so any future changes will also compile towards aarch64.

I have already added this in my own project using the [gcc_toolchain](https://github.com/aspect-build/gcc-toolchain) repo.

I guess for clang we could use https://github.com/grailbio/bazel-toolchain to configure for aarch64?

**Comment by mvukov** [comment]
Sounds good. FWIW, it looks to me that [grailbio/bazel-toolchain](https://github.com/grailbio/bazel-toolchain) is more actively developed than gcc_toolchain.
